src/cli/edit_config.zig

```diff
@@ -133,13 +133,23 @@ pub fn run(alloc: Allocator) !u8 {
     // so this is not a big deal.
     comptime assert(builtin.link_libc);
 
-    const editorZ = try alloc.dupeZ(u8, editor);
-    defer alloc.free(editorZ);
-    const pathZ = try alloc.dupeZ(u8, path);
-    defer alloc.free(pathZ);
+    var buf: std.ArrayListUnmanaged(u8) = .empty;
+    errdefer buf.deinit(alloc);
+
+    const writer = buf.writer(alloc);
+    var shellescape: internal_os.ShellEscapeWriter(std.ArrayListUnmanaged(u8).Writer) = .init(writer);
```

Comment:
I don't think we need to scape anything since it's being `exec` directly right? The shell isn't doing any argument parsing here.

Reply:
Escaping removed.

Reply:
Did you check if I was right? I wasn't 100% sure, but I'm pretty sure haha. Its worth checking.

Reply:
Argh… Yeah now that I gave myself a few seconds to think. Yeah we need to escape the pathname in case it contains spaces or some other wierdness.